editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -155,12 +155,15 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
       node?.type === TYPES.Content
         ? getContentTitle(node)
         : node?.data?.title ?? node?.data?.text ?? node?.data?.flagSet;
+    const nodeId = node?.id || "Unknown";
```

Comment:
As this is a nullable column, setting this to `node?.id || null` may make queries a little easier in Metabase?

Reply:
Yep updated !